<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeerArrayVisualizerWidgetForm</class>
 <widget class="QWidget" name="SeerArrayVisualizerWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <item>
      <widget class="QLineEdit" name="variableNameLineEdit">
       <property name="toolTip">
        <string>Variable name or expression.</string>
       </property>
       <property name="placeholderText">
        <string>variable name</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="variableAddressLineEdit">
       <property name="toolTip">
        <string>Variable address.</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>variable address</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="arrayLengthLineEdit">
       <property name="toolTip">
        <string>Total number of elements in array.</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string># elements</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="arrayOffsetLineEdit">
       <property name="toolTip">
        <string>Element offset into array. (default=0)</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>array offset</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="arrayStrideLineEdit">
       <property name="toolTip">
        <string>Element stride between values. (default=1)</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>array stride</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="arrayDisplayFormatComboBox">
       <property name="toolTip">
        <string>Array data type.</string>
       </property>
       <item>
        <property name="text">
         <string>int16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>int32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>int64</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>unt16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>uin32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>uin64</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>float32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>float64</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="refreshToolButton">
       <property name="toolTip">
        <string>Refresh the display.</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../SeeDataStructureRc.qrc">
         <normaloff>:/resources/RelaxLightIcons/view-refresh.svg</normaloff>:/resources/RelaxLightIcons/view-refresh.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="SeerArrayWidget" name="arrayTableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="25,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QZoomChartView" name="arrayChartView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>100</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="titleLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Set chart title.</string>
           </property>
           <property name="placeholderText">
            <string>chart title</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_1">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="lineRadioButton">
           <property name="text">
            <string>Line</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">lineTypeButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="splineRadioButton">
           <property name="text">
            <string>Spline</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">lineTypeButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="scatterRadioButton">
           <property name="text">
            <string>Scatter</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">lineTypeButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pointsCheckBox">
           <property name="toolTip">
            <string>Show chart data points.</string>
           </property>
           <property name="text">
            <string>Points</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="labelsCheckBox">
           <property name="toolTip">
            <string>Show chart data labels.</string>
           </property>
           <property name="text">
            <string>Labels</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SeerArrayWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">SeerArrayWidget.h</header>
  </customwidget>
  <customwidget>
   <class>QZoomChartView</class>
   <extends>QGraphicsView</extends>
   <header>QZoomChartView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../SeeDataStructureRc.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="lineTypeButtonGroup"/>
 </buttongroups>
</ui>
